### ISSUE TITLE
Manga updates not refreshing automatically and manual refresh resets read chapter count

### ISSUE THREAD

**Issue by 0ctoqus** [Aidoku/Aidoku#216]
### Description
Some mangas are not updating automatically and need to be refreshed manually. However, when I refresh a manga manually, all the already read chapters count resets.

### Steps to Reproduce
1. Open the manga app.
2. Check for manga updates in the library.
3. Observe that some mangas do not update automatically.
4. Manually refresh a manga.
5. Observe that the already read chapters count resets.
OR
5. Observe that new chapters are avalaible.

### Expected Behavior
Manga updates should refresh automatically and the already read chapters count should not reset when a manga is manually refreshed.

### Actual Behavior
Manga updates do not refresh automatically and the already read chapters count resets sometime when a manga is manually refreshed.

### Additional Information
- The issue occurs daily.
- The app is used on a regular basis.
- The issue affects the user experience negatively.

### Video of the issue
https://user-images.githubusercontent.com/42735996/230929118-70e33000-85c3-4252-a20e-49bdbbd7285e.mov

**Comment by Skittyblock** [comment]
The library refreshing might be a dead source blocking things. What sources do you have installed? I can look into that

The read chapters being reset is an Asura problem; they have anti-scraping measures that randomize chapter urls, which break's Aidoku's read history tracking. There's nothing we can do about that at the moment.

**Comment by 0ctoqus** [comment]
Hey, thank you for your reply!

The issue with the library not refreshing was caused by an incorrect setting in the library update options. I had set it to not update if I had unread chapters, which caused the problem. I managed to fix it. I'm sharing the solution here in case someone else experiences the same problem.

Regarding Asura's scan breaking and changing the chapter URLs, which breaks Aidok's read history tracking, using chapter names instead of URLs in the read history could be a possible solution.
